packages/docusaurus-theme-openapi-docs/src/theme/ApiExplorer/CodeSnippets/index.tsx

```diff
@@ -335,11 +354,13 @@ function CodeSnippets({ postman, codeSamples }: Props) {
                 defaultValue={selectedVariant}
                 lazy
               >
-                {lang.variants.map((variant) => {
+                {lang.variants.map((variant, index) => {
                   return (
                     <CodeTab
                       value={variant.toLowerCase()}
-                      label={variant.toUpperCase()}
+                      label={
+                        lang.labels ? lang.labels[index] : variant.toUpperCase()
+                      }
```

Comment:
labels might be useful for future support of different variant values and labels

Reply:
Makes sense...will these labels effectively override the defaults if passed via docusaurus config to a "generated" language?

Reply:
yes they will, but they're not right now. Right now they're just overriding the defaults for code samples

Reply:
Agreed...also, there is another issue with generated code snippet - if you try to rearrange the order it leads to some errors. Additionally, I had hoped to allow users to specify which variants they wanted to render, but that needs additional work.

Reply:
@sserrata with " rearrange the order" you mean in CodeSnippets `languageSet` or in Docusaurus Config `languageTabs`?

Reply:
@sserrata I have updated the code samples implementation to keep it split from variants, having two CodeTabs. Please let me know if that's accurate. I also though about hiding the Variants (auto generated) tabs if there are CodeSamples but I don't know if that's the path the framework wants to move forward

https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/697/commits/27f63767b73124d5a276be77019ea7c6348b5232

Reply:
> @sserrata with " rearrange the order" you mean in CodeSnippets languageSet or in Docusaurus Config languageTabs?

Docusaurus config languageTabs, although that issue has been around for some time and is not related to your changes.